login/login.c

```diff
@@ -249,24 +255,90 @@ int request_url(const uint8_t service_key[GLOME_MAX_PUBLIC_KEY_LENGTH],
   return 0;
 }
 
-int login_run(glome_login_config_t* config, const char** error_tag) {
-  assert(config != NULL);
-  if (config->options & VERBOSE) {
-    errorf(
-        "debug: options: 0x%x\n"
-        "debug: username: %s\n"
-        "debug: login: %s\n"
-        "debug: auth delay: %d seconds\n",
-        config->options, config->username, config->login_path,
-        config->auth_delay_sec);
-  }
+static void login_printf(glome_login_config_t* config, const char* format,
+                         ...) {
+  UNUSED(config);
+  va_list args;
+  va_start(args, format);
+  vprintf(format, args);
+  va_end(args);
+  fflush(NULL);
+}
+
+static void login_eprintf(glome_login_config_t* config, const char* format,
+                          ...) {
+  UNUSED(config);
+  va_list args;
+  va_start(args, format);
+  vfprintf(stderr, format, args);
+  va_end(args);
+  fflush(NULL);
+}
+
+static void login_syslog(glome_login_config_t* config, int priority,
+                         const char* format, ...) {
   if (config->options & SYSLOG) {
```

Comment:
`ops` struct is `static const` so it cannot be changed at runtime, I think this is safer. Previously the check for syslog was sprinkled in login code and I thought it would be better to just do it in one place.

Reply:
Agreed.